core/templates/pages/contributor-dashboard-page/question-badge/question-badge.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+import { AppConstants } from 'app.constants';
+
+@Component({
+  selector: 'question-badge',
+  templateUrl: './question-badge.component.html',
+  styleUrls: []
+})
+export class QuestionBadgeComponent {
+  @Input() type!: string;
+  @Input() value: number;
```

Comment:
```suggestion
  @Input() contributionCount: number;
```

Ditto for translation badge.

Reply:
Done